### PR TITLE
Updated typograpohy for thread and comment body

### PR DESCRIPTION
**Pull request by mzparacha**
## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/5297

## Description of Changes
- Updated Checkboxes to have same style as Design System Component
- Removed extra bullet to left of checkbox
- Codeblock now have monotype-1 (16px)
- Dividers have 16px spacing on top and bottom.
- Body text now have Body 01 instead of Body 02.
- Table cells have 1px border of neutral-200

## "How We Fixed It"

N/A

## Test Plan
- Create some thread/comment body with above elements and verify that it matches new changes above

## Deployment Plan
N/A

## Other Considerations
N/A

### CODE REVIEW

**Comment by masvelio** [comment]
> Lists now have blockquote line to the left.

Lists should NOT have line to the left

**Comment by mzparacha** [comment]
Should be fixed now @masvelio, plz lmk if any other issues

**Review by masvelio — APPROVED** [review]
looks good!

File: packages/commonwealth/client/styles/components/quill/quill_formatted_text.scss [hunk 1]
```diff
@@ -211,6 +218,18 @@
     position: relative;
     top: 2px;
     margin-right: 5px;
+    width: 16px;
+    height: 16px;
+    border: 1px solid #999;
```
[the remark this answers is not in the document]
Comment (mzparacha) [reply packages/commonwealth/client/styles/components/quill/quill_formatted_text.scss:220]: hm, i actually copied it from cw_checkbox.. And couldn't find a similar color in `packages/commonwealth/client/styles/mixins/colors.scss`

**Review by masvelio — APPROVED** [review]

**Review by CowMuon — APPROVED** [review]

**Pull request merged**